src/script/components/list/ConversationListCell.tsx

```diff
@@ -123,153 +111,144 @@ const ConversationListCell: React.FC<ConversationListCellProps> = ({
+            aria-label={t('conversationOptionsMenu')}
+            type="button"
+            onClick={event => {
+              event.stopPropagation();
+              rightClick(conversation, event.nativeEvent);
+            }}
+          ></button>
+          {!showJoinButton && (
+            <>
+              {cellState.icon === ConversationStatusIcon.PENDING_CONNECTION && (
+                <span
+                  className="conversation-list-cell-badge cell-badge-dark"
+                  data-uie-name="status-pending"
+                  title={t('callStatusPending')}
+                  aria-label={t('callStatusPending')}
+                >
+                  <Icon.Pending className="svg-icon" />
+                </span>
+              )}
+              {cellState.icon === ConversationStatusIcon.UNREAD_MENTION && (
+                <span
+                  className="conversation-list-cell-badge cell-badge-light"
+                  data-uie-name="status-mention"
+                  title={t('conversationStatusUnreadMention')}
+                  aria-label={t('conversationStatusUnreadMention')}
+                >
+                  <Icon.Mention className="svg-icon" />
+                </span>
+              )}
+              {cellState.icon === ConversationStatusIcon.UNREAD_REPLY && (
+                <span
+                  className="conversation-list-cell-badge cell-badge-light"
+                  data-uie-name="status-reply"
+                  title={t('conversationStatusUnreadReply')}
+                  aria-label={t('conversationStatusUnreadReply')}
+                >
+                  <Icon.Reply className="svg-icon" />
+                </span>
+              )}
+              {cellState.icon === ConversationStatusIcon.UNREAD_PING && (
+                <span
+                  className="conversation-list-cell-badge cell-badge-light"
+                  data-uie-name="status-ping"
+                  title={t('conversationStatusUnreadPing')}
+                  aria-label={t('conversationStatusUnreadPing')}
+                >
+                  <Icon.Ping className="svg-icon" />
+                </span>
+              )}
+              {cellState.icon === ConversationStatusIcon.MISSED_CALL && (
+                <span
+                  className="conversation-list-cell-badge cell-badge-light"
+                  data-uie-name="status-missed-call"
+                  title={t('callStatusMissed')}
+                  aria-label={t('callStatusMissed')}
+                >
+                  <Icon.Hangup className="svg-icon" />
+                </span>
+              )}
+              {cellState.icon === ConversationStatusIcon.MUTED && (
+                <span
+                  className="conversation-list-cell-badge cell-badge-dark conversation-muted"
+                  data-uie-name="status-silence"
+                  title={t('callStatusMutted')}
+                  aria-label={t('callStatusMutted')}
```

Comment:
typo, also update the translation string
```suggestion
                  title={t('callStatusMuted')}
                  aria-label={t('callStatusMuted')}
```

Reply:
Since this is not a new key, I'd rather not change it. It might have side effects for the translations (like in crowding). I'm not sure

Reply:
It's a new key, I've added this one in the previous PR I think it's safe to update this one, as translations to other languages were not processed. Sorry for the typo :(

Reply:
Oohhh right, then I'll update it :ok_hand:

Reply:
Fixed :)